src/main/java/org/prebid/server/bidder/huaweiads/model/response/PictureAdm.java

```diff
@@ -0,0 +1,46 @@
+package org.prebid.server.bidder.huaweiads.model.response;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class PictureAdm {
+
+    String imageTitle;
```

Comment:
Please, dont avoid empty lines between fields. Insert them and check for similar occurences.